autoconfigure/src/main/java/info/developerblog/spring/thrift/client/BeanThriftClientBeanPostProcessor.java

```diff
@@ -10,14 +13,9 @@
 import org.springframework.context.annotation.Configuration;
 import org.springframework.stereotype.Component;
 
-import java.lang.reflect.Constructor;
-import java.lang.reflect.Field;
-import java.lang.reflect.Parameter;
-
 /**
- * Created by aleksandr on 01.09.15.
+ * Created by @driver733
```

Comment:
hmmm. please do not change created by. you can add modified by or smth else instead

Reply:
That's a new file create by myself. The `Created by aleksandr on 01.09.15.` was left by mistake after copy-pasting.